cmsdb/campaigns/run3_2023_postBPix_nano_v12/data.py

```diff
@@ -0,0 +1,110 @@
+# coding: utf-8
+
+"""
+CMS datasets from the 2023 postBPix data-taking campaign
+"""
+
+import cmsdb.processes as procs
+from cmsdb.campaigns.run3_2023_postBPix_nano_v12 import campaign_run3_2023_postBPix_nano_v12 as cpn
+
+# Add era E also?
+
+#
+# Muon
+#
+cpn.add_dataset(
+    name="data_mu_d_1",
+    id=14787686,
+    is_data=True,
+    processes=[procs.data_mu],
+    keys=[
+        "/Muon0/Run2023D-22Sep2023_v2-v1/NANOAOD",  # noqa
+    ],
+    n_files=32,
+    n_events=21462916,
+    aux={
+        "era": "D",
+    },
+)
+cpn.add_dataset(
+    name="data_mu_d_2",
+    id=14786997,
+    is_data=True,
+    processes=[procs.data_mu],
+    keys=[
+        "/Muon1/Run2023D-22Sep2023_v2-v1/NANOAOD",  # noqa
+    ],
+    n_files=30,
+    n_events=21463645,
+    aux={
+        "era": "D",
+    },
+)
```

Comment:
Is there a reason that these datasets are separated? We could also merge them to a single dataset in our database, e.g. like:

```suggestion
cpn.add_dataset(
    name="data_mu_d",
    id=14787686,  # id from Muon0 dataset
    is_data=True,
    processes=[procs.data_mu],
    keys=[
        "/Muon0/Run2023D-22Sep2023_v2-v1/NANOAOD",  # noqa
        "/Muon1/Run2023D-22Sep2023_v2-v1/NANOAOD",  # noqa
    ],
    n_files=32 + 30,
    n_events=21462916 + 21463645,
    aux={
        "era": "D",
    },
)
```